test_data/javascript/clean/frequency_lists.js.simple

```diff
@@ -1,10 +1,12 @@
-# javascript/clean/frequency_lists.js: high
+# javascript/clean/frequency_lists.js: critical
```

Comment:
Should we allow any clean samples to register as critical?

Reply:
In general, I would say no - but until we can define filetype rules, I think it's OK to present the fact that the rule is overly broad.

What I would like to define for `combo/degrader/infection` is: `filetypes = "!js"`